src/tests/ftest/cart/iv/cart_iv_one_node.py

```diff
@@ -472,7 +472,8 @@ def test_cart_iv(self):
 
         ]
 
-        time.sleep(2)
+        # wait for servers to come up.
+        time.sleep(4)
```

Comment:
is this actually needed?

Reply:
This was originally set to 2 seconds.  When set to 2 seconds, I sometime saw (very seldom) 32 retries to do the fetch (connecting to the server) and then failing after that.  I never saw this when set to 4 seconds.

Reply:
This was 32 retries to attach, then it would fail.